ga4gh/datamodel/reads.py

```diff
@@ -110,22 +109,22 @@ def __init__(self, id_):
         self._readGroups.append(readGroup)
 
 
-class HtslibReadGroupSet(datamodel.PysamSanitizer, AbstractReadGroupSet):
+class HtslibReadGroupSet(AbstractReadGroupSet, datamodel.PysamDatamodelMixin):
```

Comment:
Agreed about genuine mixins.  The problem I was running into (although I forget the details) was that python was throwing a TypeError complaining about the MRO when it hit the class definition.  The takeaway I basically came away from reading up on it was the left-to-right rule.